### PR TITLE
Added note about GCC to Office-365-custom-connectors.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5858.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Office-365-custom-connectors.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Any member of a team can connect their team to popular cloud services with the c
 
 Microsoft 365 connectors can be used with both Microsoft Teams and Microsoft 365 groups, making it easier for all members stay in sync and receive relevant information quickly. Both Microsoft Teams and Exchange use the same connector model, which allows you to use the same connectors within both platforms. It is worth noting, however, that disabling connectors for the Microsoft 365 group that a team is dependent upon will disable the ability to create connectors for that team as well.
```
Comment (JohanFreelancer9) [inline Teams/Office-365-custom-connectors.md:29]:
```suggestion
Microsoft 365 connectors can be used with both Microsoft Teams and Microsoft 365 groups, making it easier for all members to stay in sync and receive relevant information quickly. Both Microsoft Teams and Exchange use the same connector model, which allows you to use the same connectors within both platforms. It is worth noting, however, that disabling connectors for the Microsoft 365 group that a team is dependent upon will disable the ability to create connectors for that team as well.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - copy looks good but please also commit the minor additional correction.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by cazawideh — APPROVED** [review]

**Pull request merged**